t/t0095-bloom.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/sh
+
+test_description='test bloom.c'
+. ./test-lib.sh
+
+test_expect_success 'get bloom filters for commit with no changes' '
+	git init &&
+	git commit --allow-empty -m "c0" &&
+	cat >expect <<-\EOF &&
+	Filter:
+	Length:0
+	Data:
+	EOF
+	test-tool bloom get_filter_for_commit "$(git rev-parse HEAD)" >actual &&
+	test_cmp expect actual 
+'
+
+test_expect_success 'get bloom filter for commit with 10 changes' '
+	rm actual &&
+	rm expect &&
+	mkdir smallFolder &&
+	cd smallFolder &&
```

Comment:
Linux people use "directory" instead of the Windows "folder". Expect pushback on that.

Instead of `cd`, do operations that point to the directory:

`echo $i >smallDir/$i`
`git add smallDir`

This avoids issues where the test fails while in the subdir and causing later tests to be in the wrong directory.

If you ever _really need_ to `cd`, then do so in a subshell using parentheses:

```
(
    cd dir &&
    do stuff...
) &&
(back in original dir)
```